ActionGame/ActionGame/Script/Header/sc_Base.h

```diff
@@ -0,0 +1,50 @@
+
+#ifndef SC_BASE_H
+#define SC_BASE_H
+
+class SceneBase {
+public:
+	// �R���X�g���N�^
+	SceneBase();
+
+	// �f�X�g���N�^
+	~SceneBase();
+
+	virtual void SceneProcessing() = 0;
```

Comment:
動詞にしましょう。
「シーンの処理を実行する」という意味のメソッドかと思いますので、 
```suggestion
	virtual void Execute() = 0;
```
等で良さそうですが、もっと良い名前がありそうなら検討してください。

Reply:
@taishi-matsumura 
180ed4c で修正しました。

`Perform` にしようか迷いましたが、調べた感じだと `Execute` の方が場面的に合ってそうなので。